packages/replay-next/components/lexical/plugins/typeahead/TypeAheadPopup.tsx

```diff
@@ -45,7 +47,15 @@ export default function TypeAheadPopUpSuspends<Item>({
   queryData: QueryData;
   updateQueryData: (queryData: QueryData | null) => void;
 }) {
-  const items = findMatches(queryData.query, queryData.queryAdditionalData);
+  let items: Item[] = [];
+  try {
+    items = findMatches(queryData.query, queryData.queryAdditionalData);
+  } catch (errorOrPromise) {
+    if (isThennable(errorOrPromise)) {
+      throw errorOrPromise;
+    }
+    console.error("Failed to find matches for the TypeAheadPopup", errorOrPromise);
+  }
 
   return (
     <Suspense>
```

Comment:
Just noticing this now, but given this component may _already_ suspend due to code completion `findMatches` function, this inner `<Suspense>` wrapper is kind of useless.

Reply:
I noticed that too but thought I must be missing something. I'll remove it.